SpotifyAPI.Example/LocalControl.cs

```diff
@@ -70,8 +85,8 @@ public void UpdateInfos()
             versionLabel.Text = status.Version.ToString();
             repeatShuffleLabel.Text = status.Repeat + @" and " + status.Shuffle;
 
-            if (status.Track != null) //Update track infos
-                UpdateTrack(status.Track);
+            //if (status.Track != null) //Update track infos
+            //    UpdateTrack(status.Track);
```

Comment:
I also made some design changes to SpotifyLocalAPI.cs and added new events. I will upload them in a few days to discuss with you. But i need to test a few things to be sure.